src/Common/Constants.ts

```diff
@@ -548,3 +548,5 @@ export const VULNERABILITIES_SORT_PRIORITY = {
     low: 4,
     unknown: 5,
 }
+
+export const IS_PLATFORM_MAC_OS = window.navigator.userAgent.toUpperCase().includes('MAC')
```

Comment:
Check if util is required instead of const, since window may not be initially present

Reply:
I tried to turn it into a function and put it in Utils.tsx but i think circular dependency caused an issue.

I built dashboard locally and used `yarn serve` and the page was not opening.